src/ECharts/Style.elm

```diff
@@ -985,6 +985,33 @@ encodeShadowStyleOption option =
             ]
 
 
+{-| describe the rich option
+-}
+type alias RichOption =
```

Comment:
Empty record?

Reply:
Can not enable the "lineHeight" without "rich" in Echart Demo, so define an empty one.